src/OpenLoco/src/Map/MapGenerator/MapGenerator.cpp

```diff
@@ -433,13 +435,108 @@ namespace OpenLoco::World::MapGenerator
         }
     }
 
+    static void updateProgress(uint8_t value);
```

Comment:
can't you just move the method up here instead?

Reply:
Heh, I just didn't want to clutter the diff with unrelated business. But it's a small method, so why not.